developer_manual/core/acceptance-tests.rst

```diff
@@ -205,14 +205,28 @@ Now that the prerequisites are satisfied, and assuming that ``$installation_path
       --database='mysql' --database-name='owncloud' --database-user='root' \
       --database-pass='' --admin-user='admin' --admin-pass='admin'
 
+Types of Acceptance Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are 3 types of acceptance tests; API, CLI and webUI.
+
+- API tests test the ownCloud public APIs.
+- CLI tests test the ``occ`` command-line commands.
+- webUI tests test the browser-based user interface.
+
+webUI tests require an additional environment to be set up.
+See the UI testing documentation for more information.
```

Comment:
@settermjd feel free to add a commit with the "magic" in the correct format to link to ``ui-testing.rst``
But actually, if this documentation is going to be superseded by "release to the wild" of the ``docs`` repo "real soon now" then bothering with links... here now is not worth it.

Reply:
As the new docs should be live soon, I'll just approve + merge this as is.